glimp/egl/surface.cc

```diff
@@ -19,6 +19,7 @@
 
 #include <utility>
 
+#include "base/logging.h"
```

Comment:
All of //glimp does not (and should not ) depend on //base, so please undo all changes in this directory.

Glimp component is expected to depend on Starboard itself only. We are actually planning to move the entire component under //starboard/shared/glimp in future.

Reply:
All changes in the //glimp were reverted